data/human/human missions.txt

```diff
@@ -4964,3 +4964,77 @@ mission "Riley"
+			`    Her smile fades away as you say that, but they nod and turn away.`
+				decline
+			label "Agree"
+			`    "Fantastic," she says, "my name is Sophie. This is my younger brother," she points at the man to her side. "Hi, I am Liam," he says, "we have been looking for a ride for some time now, we already started to worry we wouldn't get there in time," he tells you. You nod in agreement. "A lot of captains are very busy these times, you are lucky that I have some time. My name is <first> <last>." They nod in agreement and then you lead them to your spaceship.`
+				accept
+	on complete
+		conversation
+			`When you arrive at New China's spaceport, the siblings head off to the place where their fathers last will will be read out. Liam turns to you. "Captain <last>, I have transferred the payment to your account, would it be okay for you to wait here until we come back? We will need a ride home." You agree, and go to the local bar to kill some time.`
+			`    After about two hours they come back, both seem somehow excited. You flag them down from your table, and they sit down. Sophie begins talking, "Captain <last>, it turns out we will need a bit more than just a ride home. If you want to hear us out, come here again in two hours because my brother and I need to buy some things."`
+		payment 50000
+
+mission "A dead man's path II"
+	name "Following Footprints"
+	description "Bring Sophie and Liam to Tundra in the Cebalrai System."
+	passengers 2
+	source
+		planet "New China"
+	destination
+		planet "Prime"
+	waypoint "Terminus"
+	waypoint "Aspidiske"
+	waypoint "Gorvi"
+	waypoint "Enif"
+	to offer
+		has "A dead man's path I: done"
+	on offer
+		conversation
+			`They both look very happy when they see you again. "So you have decided to help us," Liam asks. "I do not know yet, maybe you should tell me first what this is about," you answer. "Of course," Sophie says, "my brother is just can not wait to get started. Maybe I should start with who our father was. We do not know much about him. He came home only occasionly, do not get me wrong, our parents were not in conflict, his passion just did not allow him to stay at one place very long. He was a captain, not unlike you. He was restless to explore even the furthest parts of the galaxy and we believe he came further than most human captains. Sometimes he brought small items back home that looked like nothing we know." She pauses for a moment, lost in memories. "Anyways," she continues, "in his testament he told us that he left something behind for us. But because he is a great fan fantasy romans, he said he would not make it easy for us. Instead he left behind a route of waypoints that we should visit before we go to the place where he left something behind. Of course we could go directly to this place, but we want to respect our fathers last will." She looks directly at you now, "And that is why we need your help. Would you help us to get to all waypoints?"`
```

Comment:
this is too big of a text wall that needs spacing also but I think that we should not make the player speak without a choice in general; FW is a bad example for that

it's even best to have a fake choice than to have the narrator speak for us especially on things like that

Reply:
will integrate some suggestion from bene-dictator to smooth this out

Reply:
fixed